src/multiselect.rs

```diff
@@ -50,6 +50,14 @@ where
         self
     }
 
+    /// Adds multiple items to the list of options.
+    pub fn items(mut self, items: Vec<(T, impl Display, impl Display)>) -> Self {
```

Comment:
@petrsapak  Right, it makes sense! It should work more universally with a slice as a type of the argument:
```rust
pub fn items(mut self, items: &[(T, impl Display, impl Display)]) -> Self;
```
Because someone could try using a static array, and vectors work fine as well with adding an ampersand "&".

Reply:
Sure thing!